python/src/trezorlib/cli/evolu.py

```diff
@@ -33,15 +33,62 @@ def cli() -> None:
 
 
 @cli.command()
+@click.option("--proof", "-p", type=str)
 @with_session
 def get_node(
-    session: "Session",
-) -> dict[str, str]:
+    session: Session,
+    proof: Optional[str] = None,
+) -> str:
     """Return the SLIP-21 node for Evolu."""
+    proof_bytes = bytes.fromhex(proof) if proof else None
+    return evolu.get_evolu_node(session, proof=proof_bytes).hex()
+
+
+@cli.command()
+@click.argument("proof", type=str)
+@click.argument("challenge", type=str)
+@click.option("--size", "-s", type=int, default=10)
+@with_session
+def evolu_sign_registration_request(
```

Comment:
Other commands don't use the `evolu_` prefix, and we are already in the evolu module, so no risk of ambiguity.
```suggestion
def sign_registration_request(
```